api.go

```diff
@@ -228,6 +229,33 @@ func (r VshardRouterCallResp) GetTyped(result []interface{}) error {
 	return nil
 }
 
+// GetTransparent decodes a response from user defined function into result.
+// The response has the same format as a direct call (without vshard router) to this function.
+// If you have some custom decoder for some lua handler's response, you can reuse your decoder using this method.
+// P.S. the maximum length of the response array is cut to 3 elements due to lua vshard storage implementation
+// see https://github.com/tarantool/vshard/blob/dfa2cc8a2aff221d5f421298851a9a229b2e0434/vshard/storage/init.lua#L3130.
+func (r VshardRouterCallResp) GetTransparent(result interface{}) error {
```

Comment:
i think it better to use upcast in GetTyped func instead new decode function
and always use buffer instead raw messages and repack it

ref: https://github.com/KaymeKaydex/go-vshard-router/issues/122